services/blockchain-app-registry/shared/constants.js

```diff
@@ -19,7 +19,10 @@ const KV_STORE_KEY = {
 	COMMIT_HASH_UNTIL_LAST_SYNC: 'commitHashUntilLastSync',
 };
 
+const NON_META_FILES = ['.gitKeep'];
```

Comment:
Do we have any other files except .gitKeep to delete?

Reply:
Please do the opposite instead! Kindly make a list of files that should be kept instead and the others not in the list can be removed.